drivers/ovsconstants.go

```diff
@@ -0,0 +1,43 @@
+package drivers
+
+const (
+	operCreateBridge oper = iota
+	operDeleteBridge
+	operCreatePort
+	operDeletePort
+)
+
+const (
```

Comment:
yeah sounds good, let's keep two blocks then.

Also `ovsOperPathPrefix` and `ovsOperPath` can be moved right after 'endpointOperPath', just to keep similar kind of consts together